### PR TITLE
Build Process Improvements - added silent option that gets rid of all std output messages

### PR DESCRIPTION
**Pull request by greenbreakfast**
@jempatel I've added a silent option that triggers the pipe to `/dev/null` in the `do_run` function.
I like the idea of being able to control the verbosity of the compile command as well as the output of any command run through `do_run`

Let me know what you think

### CODE REVIEW

**Comment by jempatel** [comment]
> @jempatel I've added a silent option that triggers the pipe to `/dev/null` in the `do_run` function. I like the idea of being able to control the verbosity of the compile command as well as the output of any command run through `do_run`
> 
> Let me know what you think

Ah nice, That makes a lot of sense :)

File: build.sh [hunk 1]
```diff
@@ -262,6 +263,7 @@ while getopts m:v:c:o:AKdpCDXVh OPT; do
 		A) ALL_PACKAGES=1 ;;
 		K) ALL_KMODS=1 ;;
 		V) VERBOSE=1;;
+		s) SILENT=1;;
```
Comment (jempatel) [inline build.sh:266]: argument support needs to be added in getopts

```bash
while getopts m:v:c:o:AKdpCDXVsh OPT; do
```

**Review by jempatel — CHANGES_REQUESTED** [review]

File: build.sh (hunk 1 above)
Comment (greenbreakfast) [reply build.sh:266]: Added to getopts

**Comment by greenbreakfast** [comment]
@jempatel I've added `s` to the getopts command, thanks for the heads up.
I'm gonna merge this in, feel free to reopen the issue if needed.

**Pull request merged**